.eslintrc.json

```
@@ -68,5 +68,8 @@
         "message": "Do not strictly check typeof undefined (NOTE: currently this rule only detects the usage of 'undefined' string literal so this could be a misfire)"
       }
     ]
+    ,
+    "@typescript-eslint/no-unused-expressions": "off",
+    "@typescript-eslint/no-require-imports": "off"
```

```suggestion
    "@typescript-eslint/no-require-imports": "off"
  },
  "overrides": [
    {
      "files": "test/**/*",
      "rules": {
        "@typescript-eslint/no-unused-expressions": "off"
      }
    }
  ]
```

What do you think about applying this rule only to test code? an "unused expression" is neither a typescript error nor a runtime error, so this is the only catch for that potentially easy mistake.

---

Neutrally.  Do you want me to?